Cargo.toml

```diff
@@ -82,7 +82,7 @@ libp2p-connection-limits = { version = "0.4.0", path = "misc/connection-limits"
 libp2p-core = { version = "0.42.0", path = "core" }
 libp2p-dcutr = { version = "0.12.0", path = "protocols/dcutr" }
 libp2p-dns = { version = "0.42.0", path = "transports/dns" }
-libp2p-floodsub = { version = "0.45.0", path = "protocols/floodsub" }
+libp2p-floodsub = { version = "0.46.0", path = "protocols/floodsub" }
```

Comment:
this doesn't seem like a breaking change, do we need a minor version bump instead of patch?

Reply:
I would assume that the introduction of a new public field in `FloodsubConfig` could be considered as one, especially given that it doesnt use the `non_exhaustive` attribute. I could be wrong though, but would have to check semver on that.

Reply:
By the way, if we are doing a breaking change if libp2p-floodsub, would you accept a PR to align the naming of the behaviour, event, and config types with the other crates, as per https://github.com/libp2p/rust-libp2p/issues/2217?

ie. rename `FloodsubConfig` to just `Config`, `Floodsub` to `Behavior` and `FloodsubEvent` to `Event`, etc.

Reply:
> By the way, if we are doing a breaking change in `libp2p-floodsub`, would you accept a PR to align the naming of types with the other crates, as per #2217?
> 
> ie. rename `FloodsubConfig` to just `Config`, `Floodsub` to `Behavior` and `FloodsubEvent` to `Event`, etc.

We can do this in a separate PR and just use an alias pointing to the new names with the alias being deprecated :)

Reply:
Cool, will open a PR for this sometime soon then :)

Reply:
that's the problem, those fields don't need to be public especially in the `FloodsubConfig`. Could you also do it in the subsequent PR @romac? 
Thanks!